server/routes/files/delete.js

```diff
@@ -28,7 +28,7 @@ module.exports = function(config, req, res, next) {
     }
 
     if(response.statusCode !== 204) {
-      res.status(response.statusCode);
+      res.status(req.status);
```

Comment:
What you had before was correct.  Look at line 34 below, which also references `response.statusCode`.